addons/point_of_sale/tools/posbox/posbox_create_image.sh

```diff
@@ -29,11 +29,22 @@ MOUNT_POINT="${__dir}/root_mount"
 OVERWRITE_FILES_BEFORE_INIT_DIR="${__dir}/overwrite_before_init"
 OVERWRITE_FILES_AFTER_INIT_DIR="${__dir}/overwrite_after_init"
 VERSION=17.0
-VERSION_IOTBOX=24.01
-REPO=https://github.com/odoo/odoo.git
+VERSION_IOTBOX=24.06
+
+
+# ask user for the branch/version
+current_branch="$(git branch --show-current)"
+read -p "Enter dev branch [${current_branch}]: " VERSION
+VERSION=${VERSION:-$current_branch}
+
+# ask user for the repository
+current_repo="$(git remote get-url origin | sed 's/.*github.com[\/:]//' | sed 's/\/odoo.git//')"
```

Comment:
note that the previous convention for naming origin in Odoo used by old timers was `odoo-dev` and `enterprise-dev` instead of `origin`
```
odoo    https://github.com/odoo/odoo.git (fetch)
odoo    you_should_not_push_on_this_repository (push)
odoo-dev        git@github.com:odoo-dev/odoo.git (fetch)
odoo-dev        git@github.com:odoo-dev/odoo.git (push)
```

Reply:
if you need the remote of the current branch, you can get it easily
```sh
#/bin/bash
branch=$(git branch --show-current)
remote=$(git config branch.$branch.remote)
echo "You are at $remote/$branch"
```